rust/libnewsboat-ffi/src/lib.rs

```diff
@@ -85,6 +85,36 @@ pub extern "C" fn rs_get_random_value(rs_max: u32 ) -> u32 {
     utils::get_random_value(rs_max)
 }
 
+#[no_mangle]
+pub extern "C" fn rs_escape_url(input: *const c_char) -> *mut c_char {
+    let rs_input = unsafe { CStr::from_ptr(input) };
+    let rs_input = rs_input.to_string_lossy().into_owned();
+
+    let result = utils::unescape_url(rs_input);
```

Comment:
I'm pretty sure this should say `utils::escape_url`.

Reply:
probably, thanks.